pub/index.html

```diff
@@ -14,11 +14,99 @@
+                    </p>
+                    <p>
+                        Cher okra gumbo tasso barbed wire catahoula football cher sa fait chaud bonjour. Food football canaille cher <a id="article-link" href="#">tasso yams</a> canaille.
+                    </p>
+                <blockquote>"WHOA, THATS RADICAL!"</blockquote>
+                    <p>Lagniappe boucherie smoked sausage tasso sac a lait praline. Mardi Gras make a roux hot sauce beer etoufee andouille. Praline bonjour smoked sausage King Cake sauce piquante okra gumbo fried catfish Thibideaux. </p>
+                    <p>Boudin barbed wire interstate mirliton hunting. Merci beaucoup Fauxcheaux canaille hunting pecan pie pecan pie Thibideaux. Sa fait chaud Thibideaux choux praline downtown. Mardi Gras Lafayette sauce piquante bayou beignet boudin interstate bbq tasso Acadiana. Barbed wire fishing Mardi Gras.</p>
+                <div class="share">
+                    <a href="#" class="fa fa-share-alt-square fa-2x"></a><span><a href="#" class="hidden">Share</a></span>
+                    <a href="#" class="fa fa-comment fa-2x"></a><a href="#" class="hidden">Comment</a>
+                    <a href="#" class="fa fa-reddit-square fa-2x"></a><a href="#" class="hidden">Reddit</a>
+                </div>    
+            </article>
+            <aside>
+                <section class="side">
+                    <img class="author author-large" src="http://lorempixel.com/output/people-q-c-300-300-3.jpg" />
+                    <h4>First Lastname</h4>
+                        <nav>
+                            <a class="author-links" href="#">Twitter</a> 
+                            <a class="author-links" href="#">Facebook</a>
+                            <a class="author-links" href="#">Instagram</a>
+                        </nav>    
+                        
+                </section>
+                    <div class="about-ad">
+                        <figure>
+                          <img class="advertise" src="http://stuffpoint.com/food/image/235785-food-fast-food-combo.png" />
+                        </figure>
+                        <figcaption><a href="">About this ad</a></figcaption>
+                    </div>
+            </aside>
+            
+        </div>
+                <nav class="pop-posts">
+                    <a class="pop-pic pic1" href="">
+                        <figcaption>Cool Post</figcaption>
+                    </a>
+                    <a class="pop-pic pic2" href="">
+                        <figcaption>Another Post</figcaption>
+                    </a>
+                    <a class="pop-pic pic3" href="">
+                        <figcaption>Longer Post Title Here</figcaption>
+                    </a>
+                    <a class="pop-pic pic4" href="">
+                        <figcaption>Extreme Kayaking</figcaption>
```

Comment:
So the [`<figcaption>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption) tag is only permissible within a `<figure>` element.